code/game/objects/structures/guillotine.dm

```diff
@@ -0,0 +1,230 @@
+	icon = 'icons/obj/guillotine.dmi'
+	icon_state = "guillotine_raised"
+	var/blade_status = GUILLOTINE_BLADE_RAISED
+	var/blade_sharpness = GUILLOTINE_BLADE_MAX_SHARP // How sharp the blade is
+	var/kill_count = 0
+	can_buckle = TRUE
+	anchored = TRUE
+	density = TRUE
+	max_buckled_mobs = 1
+	buckle_lying = FALSE
+	buckle_prevents_pull = TRUE
+	layer = ABOVE_MOB_LAYER
+
+/obj/structure/guillotine/Initialize()
+	LAZYINITLIST(buckled_mobs)
+	. = ..()
+
+/obj/structure/guillotine/examine(mob/user)
+	..()
+
+	var/msg = ""
+
+	msg += "It is [anchored ? "wrenched to the floor." : "unsecured. A wrench should fix that."]<br/>"
+
+	if (blade_status == GUILLOTINE_BLADE_RAISED)
+		msg += "The blade is raised, ready to fall, and"
+
+		if (blade_sharpness >= GUILLOTINE_DECAP_MIN_SHARP)
+			msg += " looks sharp enough to decapitate without any resistance."
+		else
+			msg += " doesn't look particularly sharp. Perhaps a whetstone can be used to sharpen it."
+	else
+		msg += "The blade is hidden inside the stocks."
+
+	if (LAZYLEN(buckled_mobs))
+		msg += "<br/>"
+		msg += "Someone appears to be strapped in. You can help them out, or you can harm them by activating the guillotine."
+
+	to_chat(user, msg)
+
+	return msg
+
+/obj/structure/guillotine/attack_hand(mob/user)
+	if (blade_status == GUILLOTINE_BLADE_MOVING)
+		return
+
+	if (blade_status == GUILLOTINE_BLADE_DROPPED)
+		blade_status = GUILLOTINE_BLADE_MOVING
+		icon_state = "guillotine_raise"
+		addtimer(CALLBACK(src, .proc/raise_blade), GUILLOTINE_ANIMATION_LENGTH)
+		return
+
+	if (blade_status == GUILLOTINE_BLADE_RAISED)
+		if (LAZYLEN(buckled_mobs))
+			if (user.a_intent == INTENT_HARM)
+				user.visible_message("<span class='warning'>[user] begins to pull the lever!</span>",
+					                 "<span class='warning'>You begin to the pull the lever.</span>")
+				if (do_after(user, GUILLOTINE_ACTIVATE_DELAY, target = src) && blade_status == GUILLOTINE_BLADE_RAISED)
+					blade_status = GUILLOTINE_BLADE_MOVING
+					icon_state = "guillotine_drop"
+					addtimer(CALLBACK(src, .proc/drop_blade), GUILLOTINE_ANIMATION_LENGTH - 2) // Minus two so we play the sound and decap faster
+			else
+				var/mob/living/carbon/human/H = buckled_mobs[1]
+
+				if (H)
+					H.regenerate_icons()
+
+				unbuckle_all_mobs()
+		else
+			blade_status = GUILLOTINE_BLADE_MOVING
+			icon_state = "guillotine_drop"
+			addtimer(CALLBACK(src, .proc/drop_blade), GUILLOTINE_ANIMATION_LENGTH)
+
+/obj/structure/guillotine/proc/raise_blade()
+	blade_status = GUILLOTINE_BLADE_RAISED
+	icon_state = "guillotine_raised"
+
+/obj/structure/guillotine/proc/drop_blade()
+	if (buckled_mobs.len && blade_sharpness)
+		var/mob/living/carbon/human/H = buckled_mobs[1]
+
+		if (H)
+			var/obj/item/bodypart/head/head = H.get_bodypart("head")
+
+			if (head)
+				playsound(src, 'sound/weapons/bladeslice.ogg', 100, 1)
+				if (blade_sharpness >= GUILLOTINE_DECAP_MIN_SHARP || head.brute_dam >= 100)
+					head.dismember()
```

Comment:
add_logs(user, H, "beheaded", src)

Reply:
That's done but where can I see these logs? It doesn't print to the chat from what I can see

Reply:
They're admin investigation logs.